### ISSUE TITLE
normative text and intent diverge on whether persistency of the page locator (within the content flow) is requested 

### ISSUE THREAD

**Issue by gundulaniemann** [w3c/wcag#1644]
In the updated version [https://w3c.github.io/wcag/understanding/fixed-reference-points.html](https://w3c.github.io/wcag/understanding/fixed-reference-points.html) of the Success Criterion there is a discrepancy between the title, the normative text and the understanding.

The normative text says:
"For web content with page break locators, a mechanism is available to navigate to each locator."

So, relying on the normative text, the Success Criterion says that there needs to be a way to navigate to page locators. 
Solutions that come into mind include table of contents, list of page numbers, among others.
The normative text does not say anything about how or where the page locators are set, nor anything about the relation page locators have when they appear in different versions of the same document (or publication).

In contrast to this, the SC title talks about 'fixed reference points'. There is no clue in the normative text what this 'fixed' might refer to or what should be fixed.

Also in contrast to the normative text, the Intent section states 'The purpose of this success criterion is to allow people using assistive technology to find references to content based on the page break locators found in the default view or printed version of a publication.'
and it also explains "The term 'on the same page' is common in English to describe when two people are talking about the same thing and agree on the content of the conversation. This success criterion makes sure that all users can literally be on the same page."
So the intent clearly states that page locators should be consistent within the content flow, while the normative text does not request this.

Normative text and understanding should always reflect each other to prevent misinterpretation.

So either the understanding should drop any mentioning of being on the same page or finding references, or the normative text should clearly state this point, for example by adding 'and each locator maintains its place in the flow of content' which was part of the normative text in an earlier version.

**Comment by alastc** [comment]
> The normative text does not say anything about how or where the page locators are set

Indeed, but that page-locators are set is a pre-condition of the SC. 

> nor anything about the relation page locators have when they appear in different versions of the same document (or publication).

That is beyond the scope of the SC.

> the SC title talks about 'fixed reference points'. There is no clue in the normative text what this 'fixed' might refer to or what should be fixed.

The title hasn't been changed, but I'm struggling to see a better title. The essense of the SC is to be able to navigate to page-break-locators. However, "Navigating pages" would be misleading, and "Page break locators" would imply adding them is the focus, when actually it is navigating to them.

> the Intent section states 'The purpose of this success criterion is to allow people using assistive technology to find references to content based on the page break locators found in the default view or printed version of a publication.'

That is the intent of the SC, I'm not seeing a problem with that? As mentioned, the existence of PBLs is a pre-requisite, but that doesn't change the intent.

>  it also explains "The term 'on the same page'...

That isn't included any more.

> adding 'and each locator maintains its place in the flow of content' 

The way page break locators are defined, this is not needed. If the PBL marks the location (e.g. start) of a page, that will maintain it's place in the flow in any relevant technology.

Overall, I'm still not seeing the misalignment. The understanding document does state there is a separate requirement (in ePub) to identify the version of the document, and meeting the SC does support people finding the correct references (pages).

**Comment by gundulaniemann** [comment]
> > nor anything about the relation page locators have when they appear in different versions of the same document (or publication).
> 
> That is beyond the scope of the SC.
> 
> > the Intent section states 'The purpose of this success criterion is to allow people using assistive technology to find references to content based on the page break locators found in the default view or printed version of a publication.'
> 
> That is the intent of the SC, I'm not seeing a problem with that? As mentioned, the existence of PBLs is a pre-requisite, but that doesn't change the intent.
> 
Well, if I see these statements together, I see a contradiction. 
One one hand, the relation page locators have when they appear in different versions of the same document is out of scope, on the other people should be able to find a reference based on the page locator found in a print version.
The latter is only possible if the page locators are related across document versions, that is, if they are consistent across document versions.

> > adding 'and each locator maintains its place in the flow of content'
> 
> The way page break locators are defined, this is not needed. If the PBL marks the location (e.g. start) of a page, that will maintain it's place in the flow in any relevant technology.
>
This way, the consistency of the PBL is hidden in the definition. An unsuspicious reader will overlook that.

> > the SC title talks about 'fixed reference points'. There is no clue in the normative text what this 'fixed' might refer to or what should be fixed.
> 
> The title hasn't been changed, but I'm struggling to see a better title. The essense of the SC is to be able to navigate to page-break-locators. However, "Navigating pages" would be misleading, and "Page break locators" would imply adding them is the focus, when actually it is navigating to them.
> 
> Perhaps "Page break navigation"?
> 
It could also be named 'navigable page locators' or 'page navigation' or 'navigable reference points'.
Yet think before renaming, we should be clear on the scope of the requirement itself.

**Comment by DavidMacDonald** [comment]
@gundulaniemann writes
> Well, if I see these statements together, I see a contradiction.
One one hand, the relation page locators have when they appear in different versions of the same document is out of scope, on the other people should be able to find a reference based on the page locator found in a print version.
The latter is only possible if the page locators are related across document versions, that is, if they are consistent across document versions.

The success criteria has had a long journey for several years. One of the goals of the ePub accessibility team in discussing this with us was certainly to have ways to find pages across print to electronic. But through much  investigation, they have agreed that this is not something we can achieve in a WCAG SC which has no jurisdiction off the web. However, they said there is value in being able to navigate to reference points and that is what the SC is focusing on.

**Comment by gundulaniemann** [comment]
> @gundulaniemann writes
> 

> The success criteria has had a long journey for several years. One of the goals of the ePub accessibility team in discussing this with us was certainly to have ways to find pages across print to electronic. But through much investigation, they have agreed that this is not something we can achieve in a WCAG SC which has no jurisdiction off the web. However, they said there is value in being able to navigate to reference points and that is what the SC is focusing on.

Thank you for the clarification.
So if the scope of the SC clearly is to have a way to navigate to page locators, I suggest to take out every statement regarding the ways to find pages across print to electronic. 
So I suggest to name the SC something like 'navigable reference points', and to take out the statements on finding a page according to a page number in print version from the intent and from the examples.
Otherwise the SC might be understood differently.

**Comment by alastc** [comment]
> So if the scope of the SC clearly is to have a way to navigate to page locators, I suggest to take out every statement regarding the ways to find pages across print to electronic.

But the purpose of using page break locators is to be able to cross reference other versions.

It's a bit like having a law about wearing seat belts. You wouldn't have a law that says "you must wear seat belts" without any reference to driving a car.

**Comment by gundulaniemann** [comment]
Well, I see it a bit differently.
To me it sounds more like the requirement in an office building, that the facility manager shall have a key for each room, including the offices. If it is worded like this, only by secondary information and additional guidance it can be known that all offices should be opened by the same key by the facility manager (while at the same time employees can open only their own office with their respective key).

If the purpose  of using page break locators is to be able to cross reference other versions, this should also be part of the normative text. Otherwise this piece of information is hidden.

**Comment by alastc** [comment]
> If the purpose of using page break locators is to be able to cross reference other versions, this should also be part of the normative text. Otherwise this piece of information is hidden.

But we cannot normatively require that people cross reference a non-web version of something. 

The normative requirement has to be exactly what we are requiring people do. Then, as is often the case, the understanding document explains the overall aim.

E.g. Consistent navigation has quite a narrow scope nominatively, but the intent "is to encourage the use of consistent presentation and layout for users who interact with repeated content within a set of Web pages and need to locate specific information or functionality more than once."

There is no normative mention of presentation or layout in the SC text, but it helps to explain the purpose of the requirement.

**Comment by alastc** [comment]
Draft response (based on the above):

<hr>

The overall aim of the SC is to support the use of page numbers as a means of navigation, particularly for publications with paper or digital versions.

However, we cannot normatively require that people cross reference a non-web version of something. The normative requirement has to be exactly what we are requiring people do. Then, as is often the case, the understanding document explains the overall aim.
